### PR TITLE
fix: add index file and package.json values to fix pnpm/typings error

### PR DESCRIPTION
**Pull request by josephfh**
Hopefully fixes the issue with pnpm detailed here: https://github.com/InsightSoftwareConsortium/xstate-lit/issues/14

### CODE REVIEW

**Comment by PaulHax** [comment]
Good stuff!  

The Node.js docs are saying `exports` is the new `main`.  https://nodejs.org/api/packages.html#main-entry-point-export

If you replace the `main` section with
```
"exports": "./dist/index.js"
``` 
does it still work?

**Comment by josephfh** [comment]
```
  "exports": "./dist/index.js",
  "typings": "./dist/index.d.ts",
  ```
  works!

File: package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "name": "xstate-lit",
   "version": "0.0.0-development",
   "description": "",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "typings": "dist/index.d.ts",
```
Comment (thewtex) [inline package.json]:
```suggestion
  "type": "module",
  "module": "dist/index.js",
  "types": "dist/index.d.ts",
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "browser": "./dist/index.js",
      "default": "./dist/index.js"
    }
  },
```

**Review by thewtex — COMMENTED** [review]
Thank you for the contribution! :pray: :sun_with_face: 

Could you please test the inline suggestion? As @PaulHax notes and as recommended in Node.js documentation, is could possibly work with more build tools.

**Comment by josephfh** [comment]
I have tested the suggested code and the following now resolves in an Astro.js project, with typings

`import {SelectorController} from 'xstate-lit'`

Tested with yarn & pnpm & npm ✅

**Comment by PaulHax** [comment]
Thanks y'all!

**Pull request merged**